Automattic/pocket-casts-ios
# Cleanup: remove bookmarks feature flag

Fixes #1305

Removes bookmark feature flag and all associated code.

## To test

Steps to test your PR.

- Start Pocket Casts
- Open a episode of you favorite podcast
- Check that you can see the bookmark option in the upper tab control
- Tap on it
- If you have a Plus/Patron account you should see your list of bookmarks
- if you don't a Plus/Patron account you should see CTA to start a free trial of Plus
- Tap on play on one episode
- Tap on the player
- See if you can see the bookmark option on the more actions menu ...
- Tap on it
- If you have a Plus/Patron account you should see your list of bookmarks
- if you don't a Plus/Patron account you should see CTA to start a free trial of Plus

## Checklist
